### PR TITLE
Update only jackson-core-databind from scala-steward

### PR DESCRIPTION
**Pull request by raboof**
The rest will follow, to avoid creating too many PR's

As discussed offline last week

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -21,8 +21,7 @@ object Dependencies {
   // needs to be inline with the aeron version
   val agronaVersion = "1.0.8"
   val nettyVersion = "3.10.6.Final"
-  val jacksonVersion = "2.10.1"
-  val jacksonDatabindVersion = "2.10.1"
+  val jacksonVersion = "2.10.0"
```
Comment (patriknw) [inline project/Dependencies.scala]: yes, we can introduce that again if they start with micro-patches of databind

File: .scala-steward.conf [hunk 2]
```diff
@@ -1,6 +1,11 @@
 updates.ignore = [
   { groupId = "com.google.protobuf", artifactId = "protobuf-java" },
   { groupId = "org.scalatest", artifactId = "scalatest" },
+  { groupId = "com.fasterxml.jackson.module", artifactId = "jackson-module-parameter-names" },
+  { groupId = "com.fasterxml.jackson.module", artifactId = "jackson-module-scala" },
+  { groupId = "com.fasterxml.jackson.core", artifactId = "jackson-annotations" },
+  { groupId = "com.fasterxml.jackson.core", artifactId = "jackson-core" },
+  { groupId = "com.fasterxml.jackson.dataformat", artifactId = "jackson-dataformat-cbor" },
```
Comment (patriknw) [inline .scala-steward.conf]: also jackson-datatype-jsr310, jackson-datatype-jdk8 ?

**Review by patriknw — APPROVED** [review]
LGTM

File: project/Dependencies.scala (hunk 1 above)
Comment (johanandren) [reply project/Dependencies.scala]: But it should be 2.10.1 no?

Comment (patriknw) [reply project/Dependencies.scala]: good catch 😄

Comment (raboof) [reply project/Dependencies.scala]: doh :hushed:

**Review by johanandren — APPROVED** [review]

**Pull request merged**